### PR TITLE
Update colour themes

### LINKED ISSUE #66: Update colour theme
@ethan-tbd and i experimented with a couple of different flavours: 

Screenshot 2024-02-15 at 11 56 23 AM

but ultimately arrived to this snazzy and all grown up approach:

Screenshot 2024-02-15 at 11 56 34 AM

text input elements will also get a small glow up: 

Screenshot 2024-02-15 at 11 56 38 AM

like my childhood piggy bank: always open for future change

### PR DESCRIPTION
**Pull request by kirahsapong**
closes #66 

sort of took some creative liberties but might be beyond the zone of sanity

Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 50 42
Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 50 45
Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 50 49
Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 50 54
Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 50 58
Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 51 05

Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 48 41
Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 49 53
Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 48 43
Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 48 48
Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 48 34
Simulator Screenshot - iPhone 15 - 2024-02-15 at 16 48 38

### CODE REVIEW

File: lib/shared/theme/color_scheme.dart [hunk 1]
```diff
@@ -1,68 +1,31 @@
...
-  onInverseSurface: Color(0xffFFEC19),
-  inversePrimary: Color(0xffFFEC19),
-  shadow: Color(0xff000000),
-  surfaceTint: Color(0xff212121),
-  outlineVariant: Color(0xff8D8D8D),
-);
+    brightness: Brightness.light,
+    primary: Color(0xff161616),
+    onPrimary: Color(0xffFFEC19),
+    secondary: Color(0xff9A1AFF),
```
Comment (ethan-tbd) [inline lib/shared/theme/color_scheme.dart:7]: how come `lightColorScheme` uses purple in `secondary` and `onSurfaceVariant`? not enough ✨color✨?

Comment (ethan-tbd) [reply lib/shared/theme/color_scheme.dart:7]: think for now we should stick with black + white + yellow in the light theme (meaning `secondary` and `surfaceVariant` would be some variation of black/dark gray) but i have a feeling the whole color theme is something we'll be coming back to throughout the year!

**Review by ethan-tbd — APPROVED** [review]
!!

**Review by wesbillman — APPROVED** [review]
Nice!!

**Pull request merged**